src/ID3Frames.js

```diff
@@ -64,12 +64,14 @@ module.exports.APIC = {
             }
 
             return new ID3FrameBuilder("APIC")
-                .appendStaticNumber(0x01, 1)
+                .appendStaticNumber(0x00, 1)
                 .appendNullTerminatedValue(mime_type)
                 .appendStaticNumber(0x03, 1)
-                .appendNullTerminatedValue(data.description, 0x01)
+                .appendStaticValue(data.description)
+                .appendStaticNumber(0x00, 1)
```

Comment:
<img width="1271" alt="스크린샷 2021-01-01 오후 3 45 30" src="https://user-images.githubusercontent.com/9072743/103434812-bfda2400-4c49-11eb-87c2-5d2018f5b09b.png">
I compared current version(v2.0.1)'s buffer with v1.0.19's buffer.
current version has one more 00 field.
Am I properly had it fixed?